packages/remix-dev/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # `@remix-run/dev`
 
+## 1.8.0
+
+### Minor Changes
+
+- Added support for a new route `meta` API to handle arrays of tags instead of an object. For details, check out the [RFC](https://github.com/remix-run/remix/discussions/4462). ([#4610](https://github.com/remix-run/remix/pull/4610))
+
+### Patch Changes
+
+- Fix missing async return in `remix dev` command ([#4660](https://github.com/remix-run/remix/pull/4660))
```

Comment:
I think this can go away, this was something we found and fixed during the prerelease, but it shouldn't have existed in 1.7.6.